src/Components/Searchbar.js

```diff
@@ -12,7 +13,9 @@ const Searchbar = () => {
   }
 
   const searchCourses = () => {
-    navigate(`/courses?name=${inputValue}`)
+    !inputValue ?
+      navigate('/courses') :
```

Comment:
This just tells it change the url depending on whether anything has been typed into the search bar.